### PR TITLE
Partial reversal of PR #141

### LINKED ISSUE #150: Remotes recognised as lamps
### Where are you using pytradfri (eg stand-alone, Home Assistant etc)
Stand-alone

### Version of pytradfri
5.3.0

### Expected behaviour
```
light_objects = [dev for dev in devices if dev.has_light_control]
```
Above code returns all lamps, but not other devices

### Actual behaviour
The code returns both lamps and remotes, but not motion sensors.

I this is probably due to the GU10 fix from #135 (PR #141). I think we should reverse this fix.

### PR DESCRIPTION
**Pull request by Lakitna**
PR #141 proved to be based on incorrect assumptions so we're rolling it back.

Closes #150

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
-VERSION = "5.3.0"
+VERSION = "5.3.1"
```
Comment (Lakitna) [inline setup.py]: I'm assuming we want to bump the version like this.

Comment (ggravlingen) [reply setup.py]: Yes 👍🏻

**Comment by ggravlingen** [comment]
Should be good to merge?

**Comment by ggravlingen** [comment]
Sorry, hit wrong button 🤦‍♂️

**Comment by Lakitna** [comment]
I know that feeling..

I believe it is.

**Comment by Lakitna** [comment]
I know that feeling..

I believe it is.

**Comment by ggravlingen** [comment]
Thank you @Lakitna!

**Comment by ggravlingen** [comment]
Pushed to pypi

**Pull request merged**